feat: update workflow and templates (#4069)
SigmaHQ/sigma: .github/PULL_REQUEST_TEMPLATE.md

A detailed description of the pull request and any additional comments or context
-->

# Example Log Event

<!--
Fill this in case of false positive fixes
-->

# Fixed Issues

<!--
Link the fixed issues here, in case your commit fixes issues with rules or code
-->

# SigmaHQ Rule Creation Conventions

> - If you PR add new rules. Please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

Reviewer: ```suggestion
- If your PR adds new rules. Please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)
```

> - If your PR adds new rules. Please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

Reviewer: ```suggestion
- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)
```
sry, was to slow
